inference-engine/tests/ie_test_utils/common_test_utils/ngraph_test_utils.hpp

```diff
@@ -13,6 +13,7 @@
 #include <ngraph/pass/pass.hpp>
 #include <ngraph/opsets/opset6.hpp>
 
+#include "details/ie_exception.hpp"
```

Comment:
can we remove this header usage at all from all sources? exception stuff is a part of ie_common.h how